protoc-gen-openapiv2/internal/genopenapi/template.go

```diff
@@ -2204,6 +2204,36 @@ func processHeaders(inputHdrs map[string]*openapi_options.Header) (openapiHeader
 	return hdrs, nil
 }
 
+func removeInternalComments(comment string) string {
+	c := []string{}
+	for len(comment) > 0 {
+		open := strings.SplitN(comment, "(--", 2)
+		ex := strings.TrimRight(open[0], " \t")
+		// Trim only one line prior to all spaces
+		switch {
+		case strings.HasSuffix(ex, "\r\n"):
+			ex = strings.TrimSuffix(ex, "\r\n")
+		case strings.HasSuffix(ex, "\n"):
+			ex = strings.TrimSuffix(ex, "\n")
+		}
+		if ex != "" {
+			c = append(c, ex)
+		}
+		if len(open) > 1 {
+			comment = open[1]
+		} else {
+			break
+		}
+		close := strings.SplitN(comment, "--)", 2)
+		if len(close) > 1 {
+			comment = close[1]
+		} else {
+			break
+		}
+	}
+	return strings.Join(c, "")
+}
```

Comment:
Mind writing a fuzz test for this? It feels like a perfect opportunity. Could potentially run a differential fuzzer against a simple regex substitution? Or just some basic sanity checking.

Reply:
Not sure if this is what you meant - but added something :-)